lib/network/modules/components/edges/bezier-edge-dynamic.ts

```diff
@@ -34,7 +34,7 @@ export class BezierEdgeDynamic extends BezierEdgeBase<Point> {
     this._boundFunction = (): void => {
       this.positionBezierNode();
     };
-    this.body.emitter.on("_repositionBezierNodes", this._boundFunction);
+    this._body.emitter.on("_repositionBezierNodes", this._boundFunction);
```

Comment:
Out of curiosity: Why the renaming from `body` to `_body`?

Reply:
We have a rule saying that private and protected members start with underscore and public members can't start with underscore (I added this rule based on how most members were named). Before the update this rule missed this member. Now it is correctly reported.

Reply:
I see! (Should have been done in a seperate PR, though)

Reply:
The rule was already set before, it just caught one more case after this update.